ui/components/app/edit-gas-fee-popover/edit-gas-fee-popover.js

```diff
@@ -54,6 +57,27 @@ const EditGasFeePopover = ({ onClose }) => {
               priorityLevel={PRIORITY_LEVELS.CUSTOM}
               onClose={onClose}
             />
+            <NetworkStatus />
+            <Typography
+              className="edit-gas-fee-popover__know-more"
+              align="center"
+              color={COLORS.UI4}
+              fontSize="12px"
+            >
+              <I18nValue
```

Comment:
Not a blocker, just curious, what was the impetus for creating this component over using the `useI18nContext` hook?